modules/http/src/main/java/org/mule/module/http/internal/listener/NoMatchingListenerRequestHandler.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.module.http.internal.listener;
+
+import org.mule.module.http.internal.domain.InputStreamHttpEntity;
+import org.mule.module.http.internal.domain.request.HttpRequestContext;
+import org.mule.module.http.internal.listener.async.HttpResponseReadyCallback;
+import org.mule.module.http.internal.listener.async.RequestHandler;
+import org.mule.module.http.internal.listener.async.ResponseStatusCallback;
+
+import java.io.ByteArrayInputStream;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class NoMatchingListenerRequestHandler implements RequestHandler
+{
+    private Logger logger = LoggerFactory.getLogger(NoMatchingListenerRequestHandler.class);
```

Comment:
Use getClass() to create the Logger, so that messages are logged with the corresponding concrete classes